src/game.cpp

```diff
@@ -321,6 +320,10 @@ void Game::input(KeyInputErrorStatus err) {
     saveState();
     stateSaved = true;
     break;
+  case CODE_RETURN_MENU:
```

Comment:
Good job using an `enum` value. _Magic values_ are not accepted as good coding practice in many projects. 
Enums / Named variables describe values' intentions better to (new) readers of a project.